graphql/types/Item.graphql

```diff
@@ -181,7 +181,7 @@ type PriceTag {
   name: String
   ratesAndBenefitsIdentifier: RatesAndBenefitsIdentifier
   rawValue: Float
-  value: Int
+  value: Float
```

Comment:
Have you considered using a BigInt scalar?

Reply:
Because I'm afraid that using float will allow a wrong usage of the API

Reply:
does graphql include bigint? I didn't know it existed haha

Reply:
I mean, if we have this scalar on our own graphql

Reply:
From my quick Google search :v it seems that it isn't natively supported, but there is a package for that, but I'm not really sure how it works

Reply:
https://www.graphql-scalars.dev/docs/scalars/big-int

Reply:
oh I see... yeah, I agree that using something like it would be better than converting to `Float` (which isn't semantically correct)